build/package_release.php

```diff
@@ -111,8 +110,10 @@
 // Post-processing - ZIP it up
 chdir(__DIR__ . '/packaging');
 
+system("rm -f ../packages/{$version}.zip ../packages/{$version}-update.zip");
+
 echo "Packaging Mautic Full Installation\n";
-system('zip -r ../packages/' . $version . '.zip addons/ app/ bin/ media/ themes/ translations/ vendor/ favicon.ico .htaccess index.php LICENSE.txt robots.txt > /dev/null');
+system('zip -r ../packages/' . $version . '.zip . -x@../excludefiles.txt');
```

Comment:
I'd suggest putting the ` > /dev/null` back at the end here unless you want the dump in your console of all the packaged files.